api/apps/api/src/modules/projects/projects.controller.ts

```diff
@@ -736,7 +737,7 @@ export class ProjectsController {
     @Param('id') projectId: string,
     @UploadedFile() shapefile: Express.Multer.File,
     @Body() body: UploadShapefileDTO,
-  ): Promise<ShapefileUploadResponse> {
+  ): Promise<GeoFeature> {
```

Comment:
shall we change @ApiOkResponse({ type: ShapefileUploadResponse }) -> @ApiOkResponse({ type: GeoFeature }) ?

Reply:
you're right, done in 53a901080da06bda7c29dbe234a4a870529fcaf8

I should probably put forward a more structured suggestion for Swagger documentation - we've been dealing with this in a bit of a copy-and-paste way without much consideration for the actual use 😢